lib/node/pl-middle-layer/src/network_check/network_check.ts

```diff
@@ -140,6 +151,18 @@ export async function checkNetwork(
     downloadFileCheck: await downloadFile(logger, client, lsDriver, uploadBlobClient, downloadClient, filePathToDownload, fileContentToDownload),
     softwareCheck: await softwareCheck(client),
     pythonSoftwareCheck: await pythonSoftware(client, 'Jack'),
+    storageToDownloadReport: await downloadFromEveryStorage(
+      logger,
+      client,
+      lsDriver,
+      downloadClient,
+      {
+        bytesLimit: ops.everyStorageBytesLimit,
+        minFileSize: ops.everyStorageMinFileSize,
+        maxFileSize: ops.everyStorageMaxFileSize,
```

Comment:
If you download small part of file using range query, you actually can choose any file, even terabytes-big.
I think we should make this upper border unlimited, because logic can fail in storage that has only big files (big raw datasets only)

Reply:
Ok, let's remove it